docs/arts-and-crafts/perler-bead-sprites.md

```diff
@@ -0,0 +1,372 @@
+* A clothes iron
+* Flat surface
+* Tweezers for placing beads on pegboard (optional)
+* A computer with internet access
+* A color printer
+* Masking tape and scissors
+
+## Instructions
+
+* Open up your favorite [MakeCode Arcade](https://arcade.makecode.com) game in the browser.
+* Click on the image of the Sprite you want to design.
+* Using the **Marquee Tool**, select the image of the Sprite.
+
+![Marquee tool in sprite editor](/static/arts-and-crafts/perler-bead-sprites/marquee.gif)
+
+* On the keyboard, press Control-C to copy the image (Ctrl+C on Windows or Command+C on Mac).
+* NOTE – you can also open your game in the JavaScript editor and copy the img image code from there.
+* In a new browser tab, open the [MakeCode Arcade Stencilr](https://arcade-stencils.glitch.me) tool.
+* Delete the existing image code in the window and paste the image from your game using the keys Control-V on the keyboard (Ctrl+V on Windows or Command+V on Mac).
+
+![Paste the image](/static/arts-and-crafts/perler-bead-sprites/paste-image.gif)
+
+* In the Filename Prefix box, instead of "strawberry", type a name in for your image.
+* Click the Generate button.
+
+![Name the image](/static/arts-and-crafts/perler-bead-sprites/filename.png)
+
+* Scroll down to the Beads section of the page and click on the bead pattern image.
+* This will download the image as a .svg file from your browser to the **Downloads** folder on your computer.
+
+![Download the SVG](/static/arts-and-crafts/perler-bead-sprites/beads-svg.png)
+
+* Open this .svg file in your browser.
+* Print the image.
+* NOTE – Make sure you print the image without any automatic scaling or skewing of the image.
+
+![Scaling the image](/static/arts-and-crafts/perler-bead-sprites/print-scale.png)
+
+* Cut out the bead pattern from the paper and tape it to the back of a clear pegboard, making sure to line up the dots to the pegs.
+
+![Pattern to cut out](/static/arts-and-crafts/perler-bead-sprites/tape-pattern.jpg)
+
+* Pick out the bead colors you will need for your sprite image and assemble them on the pegboard. If you have tweezers, you can use them in this step.
+
+![Choose and assemble the beads](/static/arts-and-crafts/perler-bead-sprites/assemble-beads.jpg)
+
+* Once you have your design in place, place the ironing paper over your bead design.
+* Apply a hot iron over the ironing paper to slowly and gently melt the beads together. Use medium heat, a dry iron, and circular motions for about 10 seconds.
+
+![Ironing the beads](/static/arts-and-crafts/perler-bead-sprites/iron-beads.jpg)
+
+* Peel away your parchment paper and voila! Your MakeCode Arcade Sprite brought to life! 😊
+
+![Choose and assemble the beads](/static/arts-and-crafts/perler-bead-sprites/bead-sprite.jpg)
+
+## Example
+
+Here’s an example of my son’s game and the two alien sprites he created from his game!
```

Comment:
@Jaqster - What do you think about changing this to not mention the PII reference?

Reply:
What PII reference?  You mean, don't include the photo?  That's totally fine - we don't need to include the example/photo of my kid.  I think the instructions are enough for them to get it.

Reply:
The photo is fine. Just not mention who he is. Just like in the WW Gauntlets. Just being over careful maybe.

Reply:
Got it, yes, good idea.  Thanks Galen!